src/components/Articles.jsx

```diff
@@ -6,10 +6,14 @@ const Articles = () => {
     const [articlesList, setArticlesList] = useState([])
     const [sortBy, setSortBy] = useState('')
     const [orderBy, setOrderBy] = useState('')
+    const [loading, setLoading] = useState(false); 
+    
     useEffect(()=>{
+        setLoading(true); 
```

Comment:
You can delete this line and set the initial loading state on line 9 to true rather than false. This will make your code more efficient than setting the loading state as false, then as soon as the page renders turning it to true (then back to false once everything is loaded).

Reply:
Done